edvart/data_types.py

```diff
@@ -179,7 +180,7 @@ def is_date(series: pd.Series) -> bool:
     if contains_numerics:
         return False
     try:
-        converted_series = pd.to_datetime(series, errors="coerce", infer_datetime_format=True)
+        converted_series = pd.to_datetime(series, errors="coerce", format="mixed")
```

Comment:
This doesn't work with `pandas==1.5.1`:
```
>>> import pandas as pd
>>> pd.__version__
'1.5.1'
>>> pd.to_datetime('20200101', errors='coerce', format='mixed')
NaT
```

Reply:
Removed `format="mixed"`, which is not supported in `<2.0`.